storybookjs/storybook
# adds error when storyshots finds no stories

Issue: https://github.com/storybooks/storybook/issues/1330

## What I did

Added an error when storyshots finds no stories. This is makes the test suite failure more obvious than the current failure of no tests being run.

## How to test

Modify `.storybook/config.js` to find no stories, or create a new project with a storyshots suite without any stories.

Is this testable with jest or storyshots?
no
Does this need a new example in the kitchen sink apps?
no
Does this need an update to the documentation?
no
If your answer is yes to any of these, please make sure to include it in your PR.
